nusslein_vsrp/latex/overview.tex

```diff
@@ -661,24 +661,24 @@ \subsection{Summary of the algorithm}
 	\State{$\pt \leftarrow p_{start}$}
 	\While{$\pt \geq p_{min}$}
 		\State Solve LP \eqref{eq:free-adaptation}
-		\If {LP is feasible} 
+		\If {LP is feasible}
 			\State $\delta \leftarrow \| \Delta t F(\bt-b)\|$
 			\If {$\delta < tol_\delta$}
 				\State $u^{n+1} \leftarrow u^n+\Delta t F \tilde b$
-				\State  \textbf{GOTO} line\,\ref{line:update} 
+				\State  \textbf{GOTO} line\,\ref{line:update}
 			\EndIf
 		\EndIf
 		\State $\pt \leftarrow \pt - 1$
 	\EndWhile
 	\State Reduce $\dt$
-	\State \textbf{GOTO} line\,\ref{line:calc_stge} 
+	\State \textbf{GOTO} line\,\ref{line:calc_stge}
 \EndIf
-\State Calculate $error$ cf. \ref{eq:Err} \label{line:update} 
+\State Estimate $error$, cf. \eqref{eq:Err} \label{line:update}
```

Comment:
cf. means "compare" so it is probably not what is meant here.

Reply:
I've changed it to "according to". Is that okay?